deploy/charts/harvester/values.yaml

```diff
@@ -221,7 +220,7 @@ service:
     ## Specify the port of HTTPs endpoint.
     ## defaults to "8443".
     ##
-    httpsPort: 8443
+    httpsPort: 443
```

Comment:
Have we changed the port?

Reply:
Yes. The default port of HTTPS is 443.

Reply:
Also, update the default value in the comment